architecture/core_objects/builds.adoc

```diff
@@ -302,19 +310,19 @@ Copy the value returned from the above command and place it into the
 ----
 ====
 
-Then, create the `*Secret*` from the *_secret.json_* file using the following
+. Create the `*Secret*` from the *_secret.json_* file using the following
```

Comment:
SourceSecrets are separate topic from DockerSecrets, thus we create another secret. You can use both, but we've decided to have each topic covered full path.